app/src/main/java/io/gnosis/safe/ui/splash/SplashActivity.kt

```diff
@@ -29,6 +30,8 @@ class SplashActivity : BaseActivity() {
 
         viewComponent().inject(this)
 
+        settingsHandler.allowTracking(this, settingsHandler.trackingAllowed)
```

Comment:
we are setting allow screenshots in BaseActivity. So this would also better fit into BaseActivity than here